.ci/pytorch/test.sh

```diff
@@ -228,13 +233,13 @@ test_python_shard() {
     exit 1
   fi
 
-  time python test/run_test.py --exclude-jit-executor --exclude-distributed-tests --shard "$1" "$NUM_TEST_SHARDS" --verbose
+  time python test/run_test.py --exclude-jit-executor --exclude-distributed-tests "$INCLUDE_CLAUSE" --shard "$1" "$NUM_TEST_SHARDS" --verbose
```

Comment:
Could you try to use `python test/run_test.py --exclude-jit-executor --exclude-distributed-tests --include $TESTS_TO_INCLUDE --verbose` here? As the correct syntax is `python test/run_test.py --exclude-jit-executor --exclude-distributed-tests --include test_torch test_cuda --verbose`, one `--include` and without quote.

The other option would be to update `run_test.py` itself to handle quoted list of tests I think.

Reply:
@huydnn I went with the shellcheck exception, and it passes lint now.